pkg/linters/no-cyrillic/no-cyrillic.go

```diff
@@ -66,7 +66,7 @@ func (o *NoCyrillic) Run(m *module.Module) (errors.LintRuleErrorsList, error) {
 	for _, fileName := range files {
 		name, _ := strings.CutPrefix(fileName, m.GetPath())
 		name = m.GetName() + ":" + name
-		if slices.Contains(o.cfg.NoCyrillicFileExcludes, name) {
+		if fsutils.StringMatchAnyMask(name, o.cfg.NoCyrillicFileExcludes) {
```

Comment:
Name will look like this: `upmeter:/hooks/.venv/dir/file.py`

We compare it with an exception like `upmeter:/hooks/. venv/**/*`

It looks like it should work. It would not be bad to make a test for this solution.